### ISSUE TITLE
Add "pause" and "resume"

### ISSUE THREAD

**Issue by mpociot** [mano-mykingdom/downloader#1]
Hi,

thanks for the great module! Is it possible to add `pause` and `resume` methods to the downloader?

Right now it's only possible to cancel the download.

Regards,
Marcel Pociot

**Comment by AppWerft** [comment]
Hi Marcel, currently it is impossible to compile because 'setLatestEventInfo' is deprecated. Do you have a runnable version?

**Comment by mpociot** [comment]
No, sorry - I switched to the NappDownloadManager package: https://github.com/viezel/NappDownloadManager

**Comment by AppWerft** [comment]
Thanks.  Surprisingly I'm in this project too.  I cannot compile, because 'sun.org.mozilla.javascript.internal.Context;' will not resolved. Wer sagt denn, das es einfach ist!
Edit: I cannot compile but the ready2use module seams to work.
